climada/hazard/base.py

```diff
@@ -850,7 +896,9 @@ def vshape(array):
                 return array.shape
 
             # Check size for read data
-            if not np.array_equal(vshape(val), vshape(default_value)):
+            if default_value is not None and (
+                not np.array_equal(vshape(val), vshape(default_value))
+            ):
```

Comment:
```suggestion
            if (default_value is not None
               and not np.array_equal(vshape(val), vshape(default_value))
            ):
```

Reply:
I thought I had this formatted with black but this was not the case. So I did it again and black wants to format it this way:
```suggestion
            if default_value is not None and not np.array_equal(
                vshape(val), vshape(default_value)
            ):
```
Can we simply agree on black? 😅